### PR TITLE
/logs route for displaying application logs implementation.

### PR DESCRIPTION
**Pull request by ubongedem78**

### CODE REVIEW

File: src/Hng.Web/Program.cs [hunk 1]
```diff
@@ -21,8 +28,25 @@
     app.UseSwaggerUI();
 }
 
-app.UseHttpsRedirection();
+app.UseExceptionHandler(appBuilder =>
```
Comment (NdubuisiJr) [inline src/Hng.Web/Program.cs:31]: @ubongedem78 good work so far. I just have 2 suggestions

- Create an Extension folder, then create a static class, call it `LoggerExtensions`, Create an extension method called `UseGlobalErrorHandler` and move this configuration into that extension method
- Check if the environment is development return the full exception not just the `"An unexpected fault happened. Try again later."` text.

**Review by NdubuisiJr — CHANGES_REQUESTED** [review]

File: src/Hng.Web/Properties/launchSettings.json [hunk 2]
```diff
@@ -1,24 +1,33 @@
 {
+  "$schema": "http://json.schemastore.org/launchsettings.json",
```
Comment (NdubuisiJr) [inline src/Hng.Web/Properties/launchSettings.json:2]: Undo all the changes in this file.

**Review by NdubuisiJr — CHANGES_REQUESTED** [review]

**Comment by NdubuisiJr** [comment]
It looks good now but the app is not building.

**Comment by NdubuisiJr** [comment]
You are missing a “using” statement for the extension method

**Comment by ubongedem78** [comment]
I forgot to push the changes, corrected it now. Thank you @NdubuisiJr.

**Review by NdubuisiJr — APPROVED** [review]

**Pull request merged**